tests/behavioural/conftest.py

```diff
@@ -98,7 +98,7 @@ def env_config(schema_name):
 
 @pytest.fixture(scope="module")
 def target_revision():
-    return "bb2a222a1d4a"
+    return "fecff1a73d7e"
 
```

Comment:
These hash targets are a little arcane for a new user, though easily explained. Quick comment might help point people in the right direction.